lib/fbe/octo.rb

```diff
@@ -77,12 +77,19 @@ def Fbe.octo(options: $options, global: $global, loog: $loog)
               backoff_factor: 2
             )
             if options.sqlite_cache
-              store = Fbe::Middleware::SqliteStore.new(options.sqlite_cache)
+              store = Fbe::Middleware::SqliteStore.new(options.sqlite_cache, Fbe::VERSION)
               loog.info(
                 "Using HTTP cache in SQLite file: #{store.path} (" \
                 "#{File.exist?(store.path) ? "#{File.size(store.path)} bytes" : 'file is absent'}" \
                 ')'
               )
+              if store.different_versions?
```

Comment:
@Yegorov it seems that this functionality belongs to `SqliteStore` class, not to its clients. The store should decide when and how its content should be cleaned up.